tests/integration_tests/test_server.py

```diff
@@ -144,6 +144,43 @@ async def test_game_info_broadcast_to_friends(lobby_server):
         await read_until_command(proto3, "game_info", timeout=1)
 
 
+@pytest.mark.parametrize("limit", (
+    (None, 1000),
+    (1500, 1700),
+    (1500, None),
+))
+@fast_forward(5)
+async def test_game_info_not_broadcast_out_of_rating_range(lobby_server, limit):
```

Comment:
Could also test explicitly that the game is broadcast to people not in range if the rating interval is set but not enforced.

Reply:
Added some unit tests for `is_visible_to_player`